wagtail/tests/demosite/migrations/0002_add_verbose_names.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('demosite', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='blogentrypagecarouselitem',
+            name='sort_order',
```

Comment:
It looks like a migration would be needed for every app that has an Orderable. Since this field doesn't appear anywhere in the U,I could we also remove the ``verbose_name`` on ``Orderable.sort_order`` then remake the migrations?